tests/test_output_json.py

```diff
@@ -56,6 +56,8 @@ def test_unsupported_schema_raises(self, sv: SchemaVersion) -> None:
     @patch('cyclonedx.model.ThisTool._version', 'TESTING')
     def test_valid(self, get_bom: Callable[[], Bom], sv: SchemaVersion, *_: Any, **__: Any) -> None:
         snapshot_name = mksname(get_bom, sv, OutputFormat.JSON)
+        if snapshot_name is None:
```

Comment:
skipping tests when right into it? naaa.... 
this is just a stupid idea.

instead, you should be ading a filtering to the `@named_data` 
something like 
```py
    @named_data(*((f'{n}-{sv.to_version()}', gb, sv)
                  for n, gb in all_get_bom_funct_valid
                  for sv in SchemaVersion
                  if sv not in UNSUPPORTED_SV \
                    and is_compatible_setup(gb, sv)
))
```

Reply:
Agreed - just not got to that yet - as was working out how many cases I'd need!